src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -1321,5 +1321,39 @@ public static void Main(string[] args)
             // since the runtime can distinguish signatures with different modopts.
             verifier.VerifyDiagnostics();
         }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.ReadonlyReferences)]
+        public void OverloadsWithDifferentParameterModifiersShouldErrorOut_Ref_RefReadOnly()
```

Comment:
Consider dropping `ShouldErrorOut`, `WillErrorOut`, etc. from test method names, for consistency with other tests.